test/contractBridge/executeProposal.js

```diff
@@ -73,23 +73,23 @@ contract('Bridge - [execute proposal]', async (accounts) => {
         await BridgeInstance.endKeygen(Helpers.mpcAddress);
     });
 
-    it ('[sanity] bridge configured on domain', async () => {
-      assert.equal(await BridgeInstance._domainID(), originDomainID)
-    });
-
     it("isProposalExecuted should be called successfully", async () => {
-      await TruffleAssert.passes(BridgeInstance.isProposalExecuted(
+        const destinationDomainID = await BridgeInstance._domainID();
+
+        assert.isFalse(await BridgeInstance.isProposalExecuted(destinationDomainID, expectedDepositNonce));
+
+        await TruffleAssert.passes(BridgeInstance.isProposalExecuted(
```

Comment:
doesn't make sense cause there is no transaction. And the above check already called the same function.